nixos/modules/programs/benchexec.nix

```diff
@@ -0,0 +1,95 @@
+{ lib
+, pkgs
+, config
+, options
+, ...
+}:
+let
+  cfg = config.programs.benchexec;
+  opt = options.programs.benchexec;
+
+  # NOTE: `x` is asserted to be of type 'string' or 'int'
+  # by `options.programs.benchexec.users.type` (see below).
+  # NOTE: If `x` is a string, existence of `config.users.users.${x}.uid`
+  # is asserted by `config.assertions` (see below).
+  # However, it is possible that this function is evaluated on a
+  # nonexistent user before the assertion is evaluated, so
+  # the case of a nonexistent user still does not throw an error,
+  # but instead uses 'nobody'.
+  uid = x: if builtins.isString x then (config.users.users.${x} or config.users.users.nobody).uid else x;
```

Comment:
Nobody should never be used

Reply:
Do you have a suggestion on how to handle this better?

Reply:
Fixed by an additional filtering in 8a4cc816e0ff